move-vm-backend/tests/move_vm.rs

```diff
@@ -132,25 +143,46 @@ fn get_resource() {
     let store = StorageMock::new();
     let vm = Mvm::new(store).unwrap();
 
+    let address = AccountAddress::from_hex_literal("0x1").unwrap();
+    let module = read_stdlib_module_bytes_from_project("basic_coin", "signer");
+    let mut gas_status = GasStatus::new_unmetered();
```

Comment:
we could instantiate this one only once and then reuse it between different `publish_module` calls

Reply:
When we know how to build a genesis block with stdlib initialization, it won't be necessary. Genesis mock will do it for us whenever we instantiate the test environment.